src/runtime_src/core/edge/drm/zocl/zocl_drv.c

```diff
@@ -944,6 +949,7 @@ static int zocl_drm_platform_probe(struct platform_device *pdev)
 			goto err_sched;
 	}
 
+	zocl_init_aie(drm);
```

Comment:
this function can fail, please do add condition to handle it.

Reply:
Also, it is better do the init during loadxclbin? We don't need to create this on any edge platform or even vck190 unless there is a AIE in xclbin?

Reply:
done